apps/website/docs/infrastructure/azure/apim/best-practices.md

```diff
@@ -0,0 +1,119 @@
+---
+sidebar_label: Best Practices
+sidebar_position: 1
+---
+
+# API Management Best Practices
+
+This documentation outlines a few best practices for software engineers using
+APIM.
+
+The goal is to:
+
+- Enhance security and confidence in API modifications by enabling testing in
+  production
+- Improve operational simplicity
+- Increase developer autonomy
+
+## Leveraging Versioning and Revisions for Safe Testing
+
+APIM provides built-in capabilities for API versioning and revisions, offering a
+structured way to manage API changes without disrupting active services.
+
+Thus, leveraging versioning and revisions is recommended for:
```

Comment:
```suggestion
Thus, leveraging revisions is recommended for:
```

Reply:
By changing the section title (as said above), I think we can keep this text as it is. This is a comprehensive list of unlocked potential by using these features. If you wish I can split them in two separate lists though

Reply:
Version aren't used for testing or debugging, but to serve specific changes to end users (API clients). I won't mix the two concepts.

Reply:
Updated